browser/net/brave_service_key_network_delegate_helper.cc

```diff
@@ -20,17 +21,18 @@ int OnBeforeStartTransaction_BraveServiceKey(
     net::HttpRequestHeaders* headers,
     const ResponseCallback& next_callback,
     std::shared_ptr<BraveRequestInfo> ctx) {
-  const std::vector<std::string> allowed_domains = {
-      kExtensionUpdaterDomain, GURL(UPDATER_DEV_ENDPOINT).host(),
-      GURL(UPDATER_PROD_ENDPOINT).host()};
+  const std::string allowed_domains[] = {
```

Comment:
let's make it as a `NoDestructor` instance. We don't need to recreate/update this array on each call.